Documentation/content/docs/vtk_react.md

```diff
@@ -0,0 +1,161 @@
+title: Using vtk.js with React
+---
+
+This is a quickstart tutorial for using vtk.js with React.
+
+## Initialize your project
+
+Use `npx create-react-app my-vtkjs-app`, `yarn create react-app my-vtkjs-app`, or `npm init react-app` to initialize your project.
+For this tutorial, we will use `yarn`.
+
+```sh
+$ yarn create react-app my-vtkjs-app
+$ cd my-vtkjs-app
+$ ls
+node_modules/  package.json  public/  README.md  src/  yarn.lock
+```
+
+Now install vtk.js as a dependency.
+
+```sh
+$ yarn add @kitware/vtk.js
+```
+
+## Using vtk.js in your app
+
+To add a minimal vtk.js example to your app, replace `src/App.js` with the following contents.
+
+```js src/App.js
+import { useState, useRef, useEffect } from 'react';
+
+import '@kitware/vtk.js/Rendering/Profiles/Geometry';
+
+import vtkFullScreenRenderWindow from '@kitware/vtk.js/Rendering/Misc/FullScreenRenderWindow';
```

Comment:
This is good, but it might be interesting to share/include a React component that map to a vtkRenderWindow...
Maybe we can bundle those in the code without importing react or what not? (Same for vue?)

Reply:
Or we could just point people to check out `react-vtk-js`. But yeah, having extra docs for that could be a useful thing down the road. For this tutorial, I'm less concerned about it, but it is an option.

Reply:
agree, for the current step it is good.

Reply:
It is just that I don't want users use the fullscreenRW in production when their goal is to have the 3D not in fullscreen.